event_sourcery/event_store.py

```diff
@@ -102,20 +101,6 @@ def publish(
         )
         self._outbox_storage_strategy.put_into_outbox(serialized_events)
 
-    @publish.register
```

Comment:
I don't think that should be removed. We still have a counterpart `append` that works with singledispatch. Wasn't this method covered by any tests...?

Reply:
This is a private method, that is not used anywhere. If it's needed it should be covered.